spack_config/toss_4_x86_64_ib_cray/spack.yaml

```diff
@@ -0,0 +1,579 @@
+spack:
+
+  config:
+    install_tree:
+      root: $spack/..
+      padded_length: 128
```

Comment:
This padding is quite annoying in the Gitlab logs. Why do you enable it?

Reply:
I did not see issues in #3 with this commented out. Maybe this is because the binaries were already generated with padding the registry. I don't like it because it makes reading the log output (in case of errors) even harder than it already is.